sequencer/src/api.rs

```diff
@@ -566,13 +590,14 @@ pub mod test_helpers {
         let options = opt(Options::with_port(port).status(Default::default()));
         let anvil = Anvil::new().spawn();
         let l1 = anvil.endpoint().parse().unwrap();
-        let _network = TestNetwork::new(
-            options,
-            [no_storage::Options; TestConfig::NUM_NODES],
-            l1,
-            None,
-        )
-        .await;
+        let network_config = TestConfigBuilder::default().l1_url(l1).build();
+        let config = TestNetworkConfigBuilder::<5, _, MockStateCatchup>::default()
+            .api_config(options)
+            .persistences([no_storage::Options; 5])
+            .network_config(network_config)
+            .catchups(std::array::from_fn(|_| MockStateCatchup::default()))
```

Comment:
Make this default too?

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1641/commits/cc8033aaa057e687ebdd75c67c26df62a1050167